bucket.go

```diff
@@ -18,6 +17,14 @@ const (
 	BucketTypeUser = BucketType(0)
 	// BucketTypeSystem is an internally created bucket that cannot be deleted/renamed.
 	BucketTypeSystem = BucketType(1)
+	// SystemBucketRetention is the time we should retain system bucket information
+	SystemBucketRetention = time.Hour * 24 * 7
```

Comment:
We have different default requirements for different system buckets. `_monitoring` should be 7 days, and `_tasks` should be 3 days.